lib/sendFetch.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require("node-fetch");
+
+const sendFetch = (path, method, body, authorization) => {
+  if (!path) return;
+  const options = {
+    method,
+    headers: {
+      "content-type": "application/json",
+    },
+  };
+  body && (options.body = JSON.stringify(body));
+  authorization && (options.headers.authorization = authorization);
```

Comment:
Does this mean.. `if (authorization) { options.headers.authorization = authorization }` ?

Reply:
Yeah